wger/core/api/views.py

```diff
@@ -70,6 +71,55 @@ def username(self, request, pk):
         return Response(UsernameSerializer(user).data)
 
 
+class UserApiRegistrationViewset(viewsets.ModelViewSet):
```

Comment:
Add comments to describe what is going on in this file

Reply:
I have commented the file kindly check out my files